File: primer-curso/programacion/unidad-02-Java&Python/PracticaExamen/AnalizadorYOrdenacion.py
numeros = [1,3,4,6,7,8,4,2,2]

opcion = int(input("¿Que orden quieres utilizar (Ascendente (1) o Descendente (2))?:"))

for i in range(0,len(numeros)):
	for j in range(i,len(numeros)):
		if (opcion == 1 and numeros[i] > numeros[j]) or (opcion ==2 and numeros[i] < numeros[j]):
			aux = numeros[i]
			numeros[i] = numeros[j]
			numeros[j] = aux

print(numeros)

File: primer-curso/programacion/unidad-02-Java&Python/PracticaExamen/AnalizadorYOrdenacion.java
import java.util.Scanner;
public class AnalizadorYOrdenacion{
	public static void main(String[] args){
		Scanner teclado = new Scanner(System.in);

		int[] numeros = {4,5,7,6,7,8,10};
		System.out.print("¿Que orden quieres utilizar (Ascendente (1) o Descendente (2))?: ");
		int opcion = teclado.nextInt();

		for(int i = 0; i < numeros.length;i++){
			for(int j = 0; j < numeros.length;j++){
				if ((opcion == 1 && numeros[i] > numeros[j]) 
			 		|| (opcion == 2 && numeros[i]< numeros[j])){
			 			int aux = numeros[i];
			 			numeros[i] = numeros[j];
			 			numeros[j] = aux;
			 		}
			}
		}
		for(int i : numeros){
			System.out.print(i+ " ");
		}

	} 
}
